src/decorators/Subscription.ts

```diff
@@ -19,15 +20,18 @@ export function Subscription(
   maybeOptions?: SubscriptionOptions,
 ): MethodDecorator {
   const { options, returnTypeFunc } = getTypeDecoratorParams(returnTypeFuncOrOptions, maybeOptions);
-  const topics = ([] as string[]).concat(options.topics || []);
+  // const topics = ([] as string[]).concat(options.topics || []);
   return (prototype, methodName) => {
     const handler = getHandlerInfo(prototype, methodName, returnTypeFunc, options);
-    if (topics.length === 0) {
+    if (!options.topics) {
```

Comment:
This check is not needed in TypeScript - when you don't pass `topics` option, it just won't compile.